img/v/veem.com.svg

```diff
@@ -0,0 +1 @@
5 4.5 0 00.48-.16 8.79 8.79 0 003.88-2.63l.15-.18a9.09 9.09 0 00.71-1 5.76 5.76 0 00.38-.69c.1-.21.19-.41.27-.62a8.37 8.37 0 00.4-1.29v-.18a9.59 9.59 0 00.17-1.69V25.41a24.35 24.35 0 00-14.44-22.14zm-193.18-.9A8.85 8.85 0 0046.51 4.8l-13.3 23.77C34.54 31 35.6 32.79 36 33.44a14.8 14.8 0 002.45 3c5.05 3.54 10.1-.2 11.71-2.18l11-19.66a8.85 8.85 0 00-2.4-12.23zm201.31 2.8a4.23 4.23 0 01.33-1.65 4.17 4.17 0 01.9-1.33 4.45 4.45 0 011.34-.88 4.27 4.27 0 013.22 0 4.45 4.45 0 011.34.88 4.14 4.14 0 01.89 1.33 4.06 4.06 0 01.34 1.65 4.21 4.21 0 01-1.23 3 4.68 4.68 0 01-1.34.89 4.37 4.37 0 01-3.22 0 4.68 4.68 0 01-1.34-.89 4.08 4.08 0 01-.9-1.27 4.17 4.17 0 01-.33-1.73zm.74 0a3.62 3.62 0 00.27 1.42 3.49 3.49 0 001.82 1.88 3.49 3.49 0 002.69 0 3.14 3.14 0 001-.76 3.22 3.22 0 00.74-1.12 3.82 3.82 0 000-2.82 3.08 3.08 0 00-.74-1.14 3.14 3.14 0 00-1-.76 3.49 3.49 0 00-2.69 0 3.29 3.29 0 00-1 .76 3.4 3.4 0 00-.72 1.14 3.58 3.58 0 00-.32 1.4zm1.85-2.46h1.84c1.13 0 1.71.46 1.71 1.4a1.19 1.19 0 01-.38 1 1.61 1.61 0 01-.91.36l1.39 2.17h-.79l-1.32-2.11h-.8v2.08h-.74zm.74 2.18h.76a4 4 0 00.49 0 1.43 1.43 0 00.41-.09.76.76 0 00.29-.24.88.88 0 00.11-.47.79.79 0 00-.1-.4.73.73 0 00-.26-.23 1.4 1.4 0 00-.35-.1h-1.35z" fill="#007fe0"/></g></g><path d="M125.56 74.76H130l2.05 12.1c.4 2.57.73 5.06 1 7.84h.14c.44-2.78.92-5.31 1.47-7.84l2.86-12.1h4.18l2.85 12.1c.55 2.49 1.1 5.06 1.54 7.84h.15c.29-2.78.66-5.31 1-7.84l2-12.1h4.14l-4.58 24h-5.57l-2.86-12.61c-.44-1.91-.59-3.6-.88-5.79h-.15c-.29 2.19-.55 3.88-1 5.79l-2.71 12.61h-5.5zM155 89.72c0-5.87 4.11-9.46 8.32-9.46 4.88 0 7.52 3.52 7.52 8.58a10.09 10.09 0 01-.22 2.2h-11.41c.44 3 2.35 4.76 5.21 4.76a7.62 7.62 0 004.18-1.24l1.46 2.71a10.42 10.42 0 01-6.23 1.9c-4.98 0-8.83-3.44-8.83-9.45zm12-1.91c0-2.64-1.24-4.18-3.59-4.18-2 0-3.81 1.47-4.18 4.18zm7.75 6.19V72.86h4.37v21.37c0 1.06.47 1.43.91 1.43a2.07 2.07 0 00.66-.08l.52 3.19a5.14 5.14 0 01-2.28.4c-3.04 0-4.18-1.97-4.18-5.17zm9.97 0V72.86h4.37v21.37c0 1.06.47 1.43.91 1.43a2.07 2.07 0 00.66-.08l.52 3.19a5.14 5.14 0 01-2.28.4c-3.04 0-4.18-1.97-4.18-5.17zM202 80.7h3.55l.33 1.91h.12a8.84 8.84 0 015.43-2.35c4.47 0 7.07 3.63 7.07 9.2 0 6.16-3.67 9.71-7.66 9.71a7.15 7.15 0 01-4.62-2l.15 3v5.61H202zm12.05 8.8c0-3.56-1.17-5.65-3.84-5.65a5.48 5.48 0 00-3.85 2v8.32a5.38 5.38 0 003.52 1.43c2.37-.02 4.2-2.07 4.2-6.1zm7.29 4.4c0-4 3.15-5.94 10.56-6.75-.08-1.83-.81-3.37-3.23-3.37a10.11 10.11 0 00-5.06 1.69l-1.53-2.86a14 14 0 017.4-2.35c4.47 0 6.74 2.79 6.74 7.81v10.67h-3.52l-.36-2h-.15a8.39 8.39 0 01-5.5 2.41 5.06 5.06 0 01-5.35-5.25zm10.56 0v-3.81c-4.91.58-6.38 1.68-6.38 3.51 0 1.54 1 2.13 2.49 2.13a5.39 5.39 0 003.89-1.83zm8.17-18.73a2.61 2.61 0 112.6 2.45 2.41 2.41 0 01-2.6-2.45zm.41 5.53h4.36v18h-4.36zm8.57 9.06c0-5.91 3.67-9.5 7.63-9.5a6.62 6.62 0 014.69 2l-.19-2.89v-6.51h4.33v25.88H262l-.33-1.95h-.15a7.48 7.48 0 01-5.05 2.38c-4.47 0-7.42-3.48-7.42-9.41zm12.13 3.84v-8.28a5.07 5.07 0 00-3.59-1.47c-2.23 0-4.1 2.13-4.1 5.83 0 3.89 1.46 5.9 4 5.9a4.53 4.53 0 003.69-1.98z" fill="#5e5e5e"/></svg>
```

Comment:
This icon is a better fit considering it will only be 32px on the website.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 61.07 56.01"><path fill="#ff8200" d="M33.07 34.6a6644.67 6644.67 0 0 1-17.2-30.67A8.62 8.62 0 0 0 3.91 1.4l-.19.12A8.85 8.85 0 0 0 1.6 13.8l21.48 38.43A8.84 8.84 0 0 0 30.58 56a8.66 8.66 0 0 0 7.5-3.73L47.13 36a8.22 8.22 0 0 1-11.33 1.73 13.32 13.32 0 0 1-2.73-3.12z"/><path fill="#007fe0" d="M57.18 1.5a8.85 8.85 0 0 0-12.25 2.43L31.63 27.7c1.33 2.43 2.4 4.22 2.8 4.87a14.8 14.8 0 0 0 2.44 3c5.05 3.54 10.1-.2 11.71-2.18l11-19.66a8.85 8.85 0 0 0-2.4-12.23z"/></svg>
```